modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/build.gradle

```diff
@@ -1,7 +1,7 @@
 dependencies {
 	compileInclude group: "com.jayway.jsonpath", name: "json-path", version: "2.2.0"
 	compileInclude group: "com.netflix.archaius", name: "archaius-core", version: "0.4.1"
-	compileInclude group: "com.netflix.hystrix", name: "hystrix-core", version: "1.5.11"
+	compileInclude group: "com.netflix.hystrix", name: "hystrix-core", version: "1.5.18"
```

Comment:
@rijema Is there any reason why you decided to update the version?

Reply:
This is the latest version of this lib, released 6 years ago. The highest usage is concentrated on this version.  I noticed that until the .13 version some adjustments were done on the code, and I considered to ensure the quality of the lib on our portal by also upgrading and add these modifications. 

https://github.com/Netflix/Hystrix/tags